Source/iOS/Device.swift

```diff
@@ -229,4 +229,161 @@ open class Device {
     }
 
 }
+
+// MARK: - Dynamic island
+extension Device {
+    static public var hasDynamicIsland: Bool {
+        switch version() {
+        case .iPhone14Pro, .iPhone14Pro_Max:
```

Comment:
I would recommend to do as, because all the past models don't have it / will ever have in future.

case .iPhone14Pro, .iPhone14Pro_Max:
return false
default:
return false

Reply:
Hey @kishorepran ! Thank you for your feedback and recommendation. But if we'll use `default` case we can face some bugs in the near future, for example with current/my implementation when some contributors add new case (for example iPhone 15 Pro) Xcode shows an error/warning that we need to add this case inside `hasDynamicIsland` computed property and we don't forget it.